core/src/main/java/io/undertow/server/DefaultByteBufferPool.java

```diff
@@ -24,11 +24,7 @@
 
 import java.lang.ref.WeakReference;
 import java.nio.ByteBuffer;
-import java.util.ArrayDeque;
```

Comment:
Error:  /home/runner/work/undertow/undertow/core/src/main/java/io/undertow/server/DefaultByteBufferPool.java:27: Using the '.*' form of import should be avoided - java.util.*. [AvoidStarImport]